app/components/createContainer/createContainerController.js

```diff
@@ -57,6 +60,7 @@ function ($scope, $state, $stateParams, Config, Info, Container, Image, Volume,
         $scope.swarm_mode = true;
```

Comment:
The swarm detection algorithm is not applied correctly here.

Use the following:

```javascript
Config.$promise.then(function (c) {
    $scope.swarm = c.swarm;
    Info.get({}, function(info) {
      if ($scope.swarm && !_.startsWith(info.ServerVersion, 'swarm')) {
        $scope.swarm_mode = true;
      }
    });

...

```